tests/test_layers.py

```diff
@@ -1,8 +1,14 @@
 import unittest
 import ddt
 import json
+import sys
+
+sys.path.append("/home/alex/fpgaconvnet-model-rebase/")
```

Comment:
I guess this one was left here not in purpose. We should probably remove that.

Reply:
removed in  commit 4ccbb5749b9d18e883ea8b6a46e9d34d1d9bffe4